core/native-bridge.ts

```diff
@@ -1038,6 +1038,7 @@ const initBridge = (w: any): void => {
       });
     };
 
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
I'm all for this, but we might want to put this in another PR.

Reply:
Ok, I did this mainly to pass the review reboot.

Reply:
Did npm run fmt do this? If so, lets keep it.

Reply:
No for `npm run fmt`, but `npm run lint` need.